frontend/src/layout/navigation/TopBar/YearInHogButton.tsx

```diff
@@ -0,0 +1,37 @@
+import './YearInHogButton.scss'
+
+import { LemonButton } from '@posthog/lemon-ui'
+import clsx from 'clsx'
+import { HeartHog } from 'lib/components/hedgehogs'
+import { XmasTree } from 'lib/lemon-ui/icons'
+import { useState } from 'react'
+
+export function YearInHogButton({ url }: { url: string | null }): JSX.Element | null {
+    const [isHovering, setIsHovering] = useState<boolean | null>(null)
+    return url ? (
+        <div className={'relative'}>
+            <HeartHog
+                width={'36'}
+                height={'36'}
+                className={clsx(
+                    'CheekyHog',
+                    isHovering && 'CheekyHog--peek',
+                    isHovering === false && 'CheekyHog--hide'
+                )}
+            />
+            {/*need to sync background color with button*/}
+            <div className={'rounded absolute top-0 left-0 w-full h-full YearInHog__mask'} />
```

Comment:
```suggestion
            <div className={'absolute top-0 left-0 w-full h-full YearInHog__mask'} />
```

related to comment above